website_react/src/components/inputs/InputCheckbox.jsx

```diff
@@ -0,0 +1,33 @@
+import { getDefault } from "./../../utils/commonUtils";
+
+const InputCheckbox = (props) => {
+
+    return (
+        <div className="form-input-div">
+            <label className="input-label">{props.title}</label>
+            <div key={props.fieldKey}>
+                    <input
+                        type='checkbox'
+                        id={props.value + props.fieldKey}
+                        name={"list" + props.fieldKey}
+                        value={props.value}
+                        defaultChecked={
+                            getDefault(props)
```

Comment:
1. `props.value || props.default_value` all of get default places. 
2. remove the `getDefault` function